CorsixTH/Lua/languages/polish.lua

```diff
@@ -2196,9 +2209,67 @@ options_window = {
   cancel = "Anuluj",
   data_location = "Lokalizacja danych",
   font_location = "Lokalizacja czcionki",
-  language = "Game language",
+  language = "Język gry",
```

Comment:
I'd suggest leaving some indication that this is the language menu in English. 

`Język gry (language)`

Everyone who switched anything to Chinese accidentally knows why.